### DISCUSSION TITLE
Github Education.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#143950] [GitHub Education]
### Hi everyone,

Hello! how long is the normal waiting time for github education i filled out the aplication on the 6th november its not the 8th, i have not seen a responce how long un till i get the pack?

**Comment by Srishti44-g** [comment]
1. GitHub Education approval can take a few days to two weeks; you're still within the normal waiting period.

2. Check your spam/junk folder for any missed emails from GitHub, just in case.

3. If you don’t receive a response after two weeks, consider reaching out to GitHub Support for an update.

- GitHub Education Support: For issues related to the GitHub Student Developer Pack, you can contact the GitHub Education team through this form:
https://support.github.com/contact/education
- GitHub Support: You can also submit a general support ticket here for assistance
https://support.github.com/request

**Comment by dinhgiaan** [comment]
I also signed up for the student package, and perhaps the time it takes to get through that package is different for each person

**Comment by lllHunterlllSV** [comment]
**Hello! @Verzitile  👋**

Thank you for reaching out! I understand how exciting it is to get access to the **GitHub Student Developer Pack**, and I’m happy to help clarify the waiting process.

Normally, **GitHub Education applications** can take up to **7-10 business days** to process. Since you applied on November 6th, it's still within the typical processing time. Here are a few things to keep in mind:

- ⏳ **Processing Time**: In general, the review process may take a bit longer during peak periods (e.g., the start of a new semester or holiday seasons). 
- 📧 **Check Your Email**: Be sure to double-check your inbox (and spam/junk folders) for any communications from GitHub Education. Sometimes, the confirmation or approval email can get caught in filters.
- 🔄 **Application Status**: If you haven’t received a response within **10 business days**, you can follow up with GitHub Support for an update on your application status.

If you still haven't received any response after this time, don’t hesitate to reach out to [GitHub Support](https://support.github.com/) for assistance. They’ll be able to provide further guidance. 😊

I hope this helps, and best of luck with your GitHub Student Pack! 🎉
